packages/apps/auth0-actions/src/redirect_to_full_registration.ts

```diff
@@ -42,32 +36,17 @@ export const onExecutePostLogin = async (
     api.redirect.sendUserTo(REGISTRATION_FORM_URL, {
       query: {
         session_token: sessionToken,
-        redirect_uri: `https://${event.request.hostname}/continue`,
       },
     });
   } catch (error) {
     console.error(error, 'Redirection to full registration form failed');
   }
 };
 
-// Handler that will be invoked when this action is resuming after an external redirect. If your
-// onExecutePostLogin function does not perform a redirect, this function can be safely ignored.
-
+// We don't need to do anything here, but Auth0 requires us to export a function with
+// this signature in order to support the login flow continuation
+// https://auth0.com/docs/customize/actions/flows-and-triggers/login-flow/redirect-with-actions#resume-the-authentication-flow
 export const onContinuePostLogin = async (
   event: Event<Auth0User>,
   api: API<Auth0User>
-) => {
-  try {
-    const payload = api.redirect.validateToken({
-      secret: event.secrets.AUTH0_PAYLOAD_SECRET,
-      tokenParameterName: 'token',
-    });
-
-    api.user.setAppMetadata(
-      'terms_and_conditions_accepted',
-      Boolean(payload.terms_and_conditions_accepted)
-    );
-  } catch (error) {
-    console.error(error, 'Validation of redirect token failed');
-  }
-};
```

Comment:
No token was being sent to the `/continue` endpoint as the user was being redirected there with a 302. I believe Auth0's [intention](https://auth0.com/docs/customize/actions/flows-and-triggers/login-flow/redirect-with-actions#send-data-on-the-front-channel) is that a machine could `POST` to the endpoint with a token if required, but we weren't doing this. Regardless, setting app_metadata isn't of much use to us because it gets reset on login.